examples/Baselines/Halite_competition/paddle/rl_trainer/controller.py

```diff
@@ -0,0 +1,622 @@
+#   Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+from config import config
+from kaggle_environments.envs.halite.helpers import *
+
+from .agent import Agent
+from .policy import *
+from .obs_parser import *
+from .model import Model
+from .replay_memory import *
+from .algorithm import *
+from .utils import is_alive
+
```

Comment:
`import *` means import all the function here, maybe it hard for reader to understand or traceback what functions actually import here, Maybe  `from xxxx import xxxxx` with the actual used functions name can be better?